src/Symfony/Component/CssSelector/Exception/ExpressionErrorException.php

```diff
@@ -19,6 +19,6 @@
  *
  * @author Jean-François Simon <jeanfrancois.simon@sensiolabs.com>
  */
-class ExpressionErrorException extends ParseException implements ExceptionInterface
+class ExpressionErrorException extends ParseException
```

Comment:
All dropped implements related to "class re-implements an interface of super(s)"